socorro/external/postgresql/base.py

```diff
@@ -87,11 +83,13 @@ def query(self, sql, params=None, error_message=None, connection=None):
                 fresh_connection = True
             # self.context.logger.debug(connection.cursor.mogrify(sql, params))
             results = execute_query_fetchall(connection, sql, params)
+            connection.commit()
```

Comment:
Isn't it the job of the transactionexecutor to worry about transactions? I forget. It's a mess in my head.